tests/ImageSharp.Tests/Common/SimdUtilsTests.cs

```diff
@@ -221,9 +191,30 @@ static void RunTest(string serialized)
             FeatureTestRunner.RunWithHwIntrinsicsFeature(
                 RunTest,
                 count,
-                HwIntrinsics.AllowAll | HwIntrinsics.DisableAVX2 | HwIntrinsics.DisableSSE41);
```

Comment:
I can't see a test for when SSE41 is disabled now. The method falls back to SSE2 when it's not available.